geomesa-ui/src/main/ui/Gruntfile.js

```diff
@@ -56,6 +56,14 @@ module.exports = function (grunt) {
                 expand: true
             }]
         },
+        build_vendorcss: {
```

Comment:
We don't want to include vendor CSS files in this way anymore. Instead add the necessary files to the main.styl file